packages/ui/src/stories/molecules/Banner.stories.ts

```diff
@@ -23,6 +28,8 @@ type BannerStory = StoryObj<typeof Banner>
 
 export const Default: BannerStory = {
   args: {
+    size: 'normal',
```

Comment:
As you put the default size as 'normal' in the component, here you don't need to repeat it.